### PR TITLE
README & info.xml

### PR DESCRIPTION
**Pull request by MariusBluem**
- Add screenshot
- Polish info.xml and README.md
- Rename "Antivirus Configuration" to "Antivirus for Files"
- Change app name to "Antivirus for Files"

@rullzer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,53 +1,41 @@
...
-* Test uploading from clients
+## What is planned
 
-## ToDo
-
-* File size limit
-* Configurations Tuneups
-* Other OS Testing
-* Look for ideas :P
+* 💻 Test uploading from clients
```
Comment (rullzer) [inline README.md]: That works as far as I can tell ;)

**Comment by rullzer** [comment]
Cool stuff thnx :)

File: appinfo/info.xml [hunk 2]
```diff
@@ -1,32 +1,52 @@
 <?xml version="1.0"?>
 <info>
```
Comment (nickvergessen) [inline appinfo/info.xml]: Use: 
```xml
<info xmlns:xsi= "http://www.w3.org/2001/XMLSchema-instance"
	  xsi:noNamespaceSchemaLocation="https://apps.nextcloud.com/schema/apps/info.xsd">
```

File: appinfo/info.xml [hunk 3]
```diff
@@ -1,32 +1,52 @@
...
 	<id>files_antivirus</id>
-	<name>Antivirus App for files</name>
-	<summary>Scan the files in your Nextcloud for viruses</summary>
-	<description>
-	This application inspects files that are uploaded to Nextcloud for viruses before they are written to the Nextcloud storage. If a file is identified as a virus, it is either logged or not uploaded to the server. The application relies on the underlying ClamAV virus scanning engine, which the admin points Nextcloud to when configuring the application.
+	<name>Antivirus for files</name>
+	<summary>An antivirus app for Nextcloud based on ClamAV</summary>
+	<description><![CDATA[`files_antivirus` is an antivirus app for Nextcloud based on ClamAV.
+
+* 🕵️‍♂️ When the user uploads a file, it's checked
```
Comment (nickvergessen) [inline appinfo/info.xml]: remove gender, it's not going to work on most browsers anyway

File: appinfo/info.xml [hunk 4]
```diff
@@ -1,32 +1,52 @@
...
+	
 	<types>
 		<filesystem/>
 	</types>
+	
+	<category>security</category>
 	<category>tools</category>
+	
+	<documentation>
+		<admin>https://docs.nextcloud.com/server/13/admin_manual/configuration_server/antivirus_configuration.html</admin>
```
Comment (nickvergessen) [inline appinfo/info.xml]: this should use a redirect instead: https://github.com/nextcloud/documentation/blob/master/go.php#L7

Then it doe snot need changing on each nextcloud version update

File: appinfo/info.xml [hunk 5]
```diff
@@ -1,32 +1,52 @@
...
+	</documentation>
 	<bugs>https://github.com/nextcloud/files_antivirus/issues</bugs>
 	<repository type="git">http://github.com/nextcloud/files_antivirus.git</repository>
+	
+	<screenshot>https://raw.githubusercontent.com/nextcloud/files_antivirus/readme-info-xml-polishing/screenshots/1.png</screenshot>
```
Comment (nickvergessen) [inline appinfo/info.xml]: should be master instead of `readme-info-xml-polishing`

File: appinfo/info.xml [hunk 6]
```diff
@@ -1,32 +1,52 @@
 <?xml version="1.0"?>
 <info>
 	<id>files_antivirus</id>
-	<name>Antivirus App for files</name>
-	<summary>Scan the files in your Nextcloud for viruses</summary>
-	<description>
-	This application inspects files that are uploaded to Nextcloud for viruses before they are written to the Nextcloud storage. If a file is identified as a virus, it is either logged or not uploaded to the server. The application relies on the underlying ClamAV virus scanning engine, which the admin points Nextcloud to when configuring the application.
+	<name>Antivirus for files</name>
+	<summary>An antivirus app for Nextcloud based on ClamAV</summary>
+	<description><![CDATA[`files_antivirus` is an antivirus app for Nextcloud based on ClamAV.
```
Comment (nickvergessen) [inline appinfo/info.xml]: I wouldn't use the app id to describe the app.

**Comment by MariusBluem** [comment]
Please check & merge: https://github.com/nextcloud/documentation/pull/806

File: appinfo/info.xml [hunk 7]
```diff
@@ -12,9 +12,9 @@
 
 This application inspects files that are uploaded to Nextcloud for viruses before they are written to the Nextcloud storage. If a file is identified as a virus, it is either logged or not uploaded to the server. The application relies on the underlying ClamAV virus scanning engine, which the admin points Nextcloud to when configuring the application.
 For this app to be effective, the ClamAV virus definitions should be kept up to date. Also note that enabling this app will impact system performance as additional processing is required for every upload. More information is available in the Antivirus documentation.
-	]]></description>
```
Comment (nickvergessen) [inline appinfo/info.xml]: Nop!? this breaks!

**Comment by rullzer** [comment]
Fixed it up and squashed it all (15 commits for this is really to much)

**Pull request merged**